AEZ314.md

```diff
@@ -0,0 +1,47 @@
+---
+name: "@AEZ314"
+project: "CO2 Laser Cutter Project"
+---
+
+# CO2 Laser Cutter Project
+![image](https://user-images.githubusercontent.com/60713428/208343282-9c3a7b07-211c-4495-b779-816ec21fa3bc.png)
+
+## Summary
+
+Until now, I've always been on the digital and abstract side of the coding world. This was partially due to interest but also due to lack of funding at my school. Actually, I already had this plan to build a CO2 Laser Cutter before I heard about Winter Hardware Wonderland, but it was going to put me in financially tight spot, so I'm really happy that you guys are helping people like me out --thank you.
+
+By building this project, I'll gain my first experience in the hardware world and possibly come up with new project ideas later. After I actually build this project, I want to explore how I can connect the "digital side" with the "hardware side". Perhaps, I can build a website or mobile app that will control the laser cutter. Also, I have two friends that are greatly interested in this project. They are less experienced with coding than me, but I think this is better as this will be their first experience.
+
+## Plan
+
+I was a bit worried whether I could pull this project off, but after doing a lot of research, I know that I can complete it. So here'd the plan:
+
+1:
+Model the body of the laser cutter on Fusion360 so that I know the amount of materials -aluminum frames etc.- I need. I've already done this! :D
+
+2:
+Obtain the materials. Duh XD
+
+3:
+Assemble the materials, following the 3D model. This is fairly simple, but I curated a list of Reddit posts about the intracies of working with V Slots (aluminum profiles) in case I miss something out.
+
+4:
+Review all safety features, put on goggles, and make sure the machine is alive!
+
+5:
+Set up the G-Code (the communication protocol used by CNCs) sender software. At this point the software will send certain G-Code to the machine and ask me to verify what it actually does by doing physical measurements then it will callibrate itself according to the measurements.
+
+6:
+Voila, we have a CO2 laser cutter!
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| CO2 Tube | https://www.abplaser.com/hy-dy-serisi-reci-guc-kaynagi?search=t%C3%BCp&description=true | $130  |
+| Stepper Motors | https://www.amazon.com.tr/dp/B07KZL4XCL/ref=asc_df_B07KZL4XCL1661961600000/?creative=380333&creativeASIN=B07KZL4XCL&linkCode=asn | $31.69 |
+| Lens/Mirror/Nozzle Set | https://www.amazon.com.tr/Cloudray-K4060-Grav%C3%BCr-Kesici-Kafas%C4%B1/dp/B0B5D26BRX/ref=sr_1_19?__mk_tr_TR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=9XMA67OD6O0Q&keywords=co2%2Blazer%2Bayna%2Blens%2Bset&qid=1671421766&sprefix=co2%2Blazer%2Bayna%2Blens%2Bse%2Caps%2C168&sr=8-19&th=1 | $84.51  |
+
+| Total           |                                       | $246.2 |
```

Comment:
Have you factored shipping costs / taxes in your total?

Reply:
Yes. The Taxes are included in the price, and the shipping is free for purchases exceeding a certain price threshold, which these items do.